wolfcrypt/src/chacha.c

```diff
@@ -18,15 +18,12 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
-
 /*
- *  based from
```

Comment:
Can you please retain the "based from" info?

Reply:
@dgarske, Done